rma_purchase/wizards/rma_add_purchase.py

```diff
@@ -75,8 +75,8 @@ def _prepare_rma_line_from_po_line(self, line):
             {'in_route_id': operation.in_route_id.id or route,
              'out_route_id': operation.out_route_id.id or route,
              'receipt_policy': operation.receipt_policy,
-             'location_id': operation.location_id.id or \
-                            self.env.ref('stock.stock_location_stock').id,
+             'location_id': operation.location_id.id or
+                self.env.ref('stock.stock_location_stock').id,
```

Comment:
I don't think using demo data like this is a good idea...

Reply:
I agree, remove that